CHANGELOG-PENDING.md

```diff
@@ -0,0 +1,21 @@
+# Unreleased Changes
```

Comment:
I'm in favor of getting rid of the pending change log for the app for now, as I don't think keeping track of two is worth it. Open to keeping it, or introducing it later though.